bokeh/core/_templates/doc_js.js

```diff
@@ -1,4 +1,17 @@
 var docs_json = {{ docs_json }};
 var render_items = {{ render_items }};
 
-Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+function do_embed() {
+    Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+}
+
+if (typeof Bokeh != 'undefined') {
+    do_embed()
+}
+else {
+    if (typeof window._bokeh_todos === 'undefined') {
+        console.log("Bokeh is not yet loaded, deferring call to embed_items")
+        window._bokeh_todos = []
+    }
+    window._bokeh_todos.push(do_embed)
+}
```

Comment:
If we make notebook_div.html take a JS string instead of a script tag as `plot_script`, then it would get this docs_js.js file as just JS with no `<script>`, which would mean the above deferral could go in notebook_div.html instead of in this file. In turn, that would mean that for the release we aren't putting autoload/server/standalone-html at risk because they should be unaffected.